platform.md

```diff
@@ -179,3 +198,37 @@ group = [
 Where:
 
 - Both `id` and `version` MUST be present for each buildpack object in a group.
+
+### source-metadata.toml (TOML)
+
+```toml
+[source]
+type = "<source type>"
+
+[source.version]
+
+[source.metadata]
+
+```
+
+### io.buildpacks.project OCI Image label (JSON)
```

Comment:
We don't currently define any of the JSON labels in the spec. We define the TOML, and indicate when it's represented in JSON. We can change that, but for now, could we define this structure in the TOML definition above?

Reply:
Thanks @sclevine - we've removed this section in the latest commit.